webapp/src/components/CreateIcon.jsx

```diff
@@ -9,7 +9,7 @@ export default function CreateIcon({
   innerIconCustomStyles,
 }) {
   return (
-    <div>
+    <button className="btn-icon">
       <div
         className={iconVariant}
         style={{
```

Comment:
Convert inline styling to class-based. Check and make changes for all the usages within this file.